adafruit_bitmapsaver.py

```diff
@@ -91,6 +91,7 @@ def _write_pixels(output_file, pixel_source, palette):
                     color >>= 8
                     buffer_index += 1
         else:
+            result_buffer = bytearray(2048)
```

Comment:
The `result_buffer` line at Line 82 could be removed as it is no longer needed.

Reply:
Thanks for the review @lesamouraipourpre. I removed that unused copy of the variable with latest commit